roles/kokonet-setup/templates/ifcfg-eth1.j2

```diff
@@ -0,0 +1,18 @@
+DEVICE=eth1
```

Comment:
Can you open an issue to move this to NetworkManager instead? I'd like to get rid of our sysconfig stuff because it's harder to manage programatically.

Reply:
I'd like to keep it to match with ipv6-setup, otherwise we do have inconsistent playbook...